docker-compose.yaml

```diff
@@ -0,0 +1,119 @@
+version: "3.7"
+
+# docker build ./ -t pyspider:latest
+
+services:
+  rabbitmq:
+    image: rabbitmq:alpine
+    container_name: rabbitmq
+    networks:
+      - pyspider
+    command: rabbitmq-server
+  couchdb:
+    image: couchdb:latest
+    container_name: couchdb
+    networks:
+      - pyspider
+    ports:
+      - "5984:5984"
+    env_file: .env
+
+  #mysql:
+  #  image: mysql:latest
+  #  container_name: mysql
+  #  volumes:
+  #    - /tmp:/var/lib/mysql
+  #  environment:
+  #    - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+  #  networks:
+  #    - pyspider
+  #  env_file: .env
+  phantomjs:
+    image: pyspider:latest
+    container_name: phantomjs
+    networks:
+      - pyspider
+    env_file: .env
+    volumes:
+      - /Users/Keith/Documents/Projects/python/python_projects/pyspider/pyspider/config_example.json:/opt/pyspider/config.json
```

Comment:
You may want to remove your personal information. You could use a default config in docker and add some comment here to show how to replace them.

Reply:
Thanks. I forgot about this. I've replace it with a '/path/to/dir/' and added a comment at the top.